templates/ministerial/ministerial-doc-en.html

```diff
@@ -0,0 +1,56 @@
+---
+title: Documentation about Ministerial profile page template
+description: Documentation and working examples for the Ministerial profile tempalte.
+language: en
+altLangPage: ministerial-fr.html
+dateModified: 2021-09-16
+share: true
+---
+
+<div class="wb-prettify all-pre hide"></div>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+
+	<dt>Type</dt>
+	<dd>Canada.ca template</dd>
+
+	<dt>Last review</dt>
+	<dd>2021-09-15</dd>
+
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+
+	<dt>Specification</dt>
+	<dd><a href="https://design.canada.ca/mandatory-templates/ministerial-profile-pages-draft.html">Ministerial profile pages - Canada.ca mandatory template</a></dd>
+
+	<dt>Template version</dt>
+	<dd>Version 2.0</dd>
+</dl>
+
+<h2>Working example</h2>
+
+<h3>Version 2.0</h3>
+
+<ul>
+	<li><a href="ministerial-en.html">Working example</a></li>
+	<li><a href="ministerial-fr.html" lang="fr">Exemple pratique</a></li>
+</ul>
+
+<h3>Version 1.0</h3>
+
+<ul>
+	<li>Initial template</li>
+</ul>
+
+<h2>Latest changes</h2>
+<ul>
+	<li>replaced Latest, Recent announcements, and Gallery with a flexible Features section </li>
+	<li>removed the social media feed and added an optional Follow me section instead</li>
+	<li>simplified and streamlined the guidance </li>
+	<li>clarified how to display ministers, parliamentary secretaries and heads of organizations</li>
+</ul>
```

Comment:
This shoud be the outcome, and maybe it is clearer if you replace "Recent hanges" by "Change history":
```suggestion
<ul>
	<li><a href="ministerial-en.html">Working example</a></li>
	<li><a href="ministerial-fr.html" lang="fr">Exemple pratique</a></li>
</ul>

<h2>Change history</h2>

<h3>Version 2.0</h3>
<ul>
	<li>replaced Latest, Recent announcements, and Gallery with a flexible Features section </li>
	<li>removed the social media feed and added an optional Follow me section instead</li>
	<li>simplified and streamlined the guidance </li>
	<li>clarified how to display ministers, parliamentary secretaries and heads of organizations</li>
</ul>

<h3>Version 1.0</h3>
<ul>
	<li>Initial template</li>
</ul>
```